source/_integrations/signal_messenger.markdown

```diff
@@ -176,19 +188,23 @@ actions:
       message: "Message received!"
 ```
 
-**NOTE** If the parameter `mode` is set to `json-rpc`, then you can use [signal-api-receiver](https://github.com/kalbasit/signal-api-receiver) to receive from Signal as follows:
-
-```yaml
-- resource: "http://127.0.0.1:8105/receive/pop"
-  sensor:
-    - name: "Signal message received"
-      value_template: >
-        {{ value_json['envelope']['dataMessage']['message'] }}
-      json_attributes_path: envelope
-      json_attributes:
-        - source
-        - sourceNumber
-        - sourceUuid
-        - sourceDevice
-        - timestamp
-```
+**NOTE** If the parameter `mode` is set to `json-rpc`, then you can use one of these solutions:
+
+1. employ [signal-api-receiver](https://github.com/kalbasit/signal-api-receiver) to receive from Signal as follows:
+   ```yaml
+   - resource: "http://127.0.0.1:8105/receive/pop"
+     sensor:
+       - name: "Signal message received"
+         value_template: >
+           {{ value_json['envelope']['dataMessage']['message'] }}
+         json_attributes_path: envelope
+         json_attributes:
+           - source
+           - sourceNumber
+           - sourceUuid
+           - sourceDevice
+           - timestamp
+   ```
+2. use a Python script with AppDaemon like
+   [described here](https://community.home-assistant.io/t/signal-messenger-using-received-message/858118/2)
+   (this has the advantage that no extra Docker container is needed, everything is inside Home Assistant)
```

Comment:
this seems to be unrelated to the recipient change

Reply:
lets move it to a separate PR then please, so we keep this one focused on one thing only.